.github/workflows/build_all.yml

```diff
@@ -103,15 +103,15 @@ jobs:
         shell: bash
     steps:
     - uses: actions/checkout@v2
-    - name: cache install
-      id: cache-install
-      uses: actions/cache@v2
-      with:
-        path: |
-          C:\Program Files (x86)\Intel\oneAPI\compiler
-          C:\Program Files (x86)\Intel\oneAPI\tbb
-          C:\Windows\System32\OpenCL.dll
-        key: install-${{ env.WINDOWS_BASEKIT_URL }}-${{ env.WINDOWS_DPCPP_COMPONENTS }}-compiler-tbb-opencl-${{ hashFiles('**/scripts/cache_exclude_windows.sh') }}
+#    - name: cache install
+#      id: cache-install
+#      uses: actions/cache@v2
+#      with:
+#        path: |
+#          C:\Program Files (x86)\Intel\oneAPI\compiler
+#          C:\Program Files (x86)\Intel\oneAPI\tbb
+#          C:\Windows\System32\OpenCL.dll
+#        key: install-${{ env.WINDOWS_BASEKIT_URL }}-${{ env.WINDOWS_DPCPP_COMPONENTS }}-compiler-tbb-opencl-${{ hashFiles('**/scripts/cache_exclude_windows.sh') }}
     - name: install
       if: steps.cache-install.outputs.cache-hit != 'true'
```

Comment:
This 'if' dependency on the cache-install step should be commented if the step is commented